### ISSUE TITLE
Give indication of status in report preview

### ISSUE THREAD

**Issue by puneetlath** [Expensify/App#32991]
When a report is submitted and awaiting approval, the report preview in the workspace chat doesn't give any indication of the current "status" of the report. It just looks like this: 

![image](https://github.com/Expensify/App/assets/1371996/c43cd4b1-170e-45cf-bcd4-725d140ca979)

If you click into the report, you see a "next steps" banner which tells you what you are waiting on, but ideally, the report preview itself would give some indication. 

<img width="805" alt="image" src="https://github.com/Expensify/App/assets/1371996/f7c88755-c18b-43f7-acf0-06d63f17ca1a">

Some ideas that were discussed in [this thread](https://expensify.slack.com/archives/C02MW39LT9N/p1702083759703419):
- showing the next steps on the report preview
-  show the report status on the report preview (e.g. draft/submitted/paid/etc)
- show a progress bar on the report preview

We'd love to do a design exploration to explore these options (or any other ideas).

**Comment by dannymcclain** [comment]
I've started generating some ideas for this. I will share something early next week.

**Comment by JmillsExpensify** [comment]
Super looking forward to this. Thanks @dannymcclain!

**Comment by dannymcclain** [comment]
[Posted some options/threads in Slack!](https://expensify.slack.com/archives/C02MW39LT9N/p1702914391343629)

**Comment by dannymcclain** [comment]
Lots of good stuff in Slack, but I don't know that a firm decision/direction has been decided. Seems like a lot of us are liking the "bottom state bar" idea.

I'm about to be OOO for winter break, so I'm going to drop the Figma link here in case anyone wants it while I'm out.

https://www.figma.com/file/IxQBCHBtgQJss52s54LyLq/Wave-6%3A-Workspace-Expense-Chats?type=design&node-id=173-43247&mode=design&t=KDfSHFjBKo9lFhI3-11

**Comment by dannymcclain** [comment]
Going to try to get a firm direction on this early this week.

**Comment by dannymcclain** [comment]
[Posted in Slack with some updated ideas.](https://expensify.slack.com/archives/C02MW39LT9N/p1704387969938179)

**Comment by dannymcclain** [comment]
[Tons of conversation in Slack.](https://expensify.slack.com/archives/C02MW39LT9N/p1704387969938179) I've made my recommendation to the channel to hopefully establish a clear path forward for this 🤞

**Comment by dannymcclain** [comment]
Ok we're doing it! This will be the V1 option we move forward with. [Figma file is here.](https://www.figma.com/file/IxQBCHBtgQJss52s54LyLq/Wave-6%3A-Workspace-Expense-Chats?type=design&node-id=136-17739&mode=design&t=UuXpiihlzYAd6Rph-11) Let me know if anything else is needed from me throughout the implementation.

![image](https://github.com/Expensify/App/assets/7516624/d1c91557-e4bd-4336-8fb5-3df63e7b1b19)
![image](https://github.com/Expensify/App/assets/7516624/a5b62af9-c04e-4296-a3cf-397bde2b3a3b)

**Comment by dannymcclain** [comment]
Not overdue Melvin.

@puneetlath We've reached a decision for this design, but I'm honestly not sure what the next steps are for this effort.

**Comment by dannymcclain** [comment]
@JmillsExpensify I would love to simplify the next steps in the process of accomplishing this because it makes a lot of sense for them to be harmonious.

We may also want to consider what impacts that has on our `scanning, pending, hold` pattern since they're all currently using the same styles as the current "Next Steps" implementation, but I suppose we could tackle that as a separate effort.

**Comment by JmillsExpensify** [comment]
Great point, I like considering those at the very least.

**Comment by dannymcclain** [comment]
Not really overdue Melvin. I'm working on a `#whatsnext` post for this so we can roll this, as well as the simplified next steps work, into a proper design doc/workflow.

**Comment by dannymcclain** [comment]
[WN Post here.](https://expensify.slack.com/archives/CC7NECV4L/p1706718342306509)

Based on the reactions, I think we should keep moving this one forward. @JmillsExpensify do you think we should close this issue in favor of a proper design doc tracking issue?

**Comment by dannymcclain** [comment]
Not overdue Melvin.

**Comment by dannymcclain** [comment]
It's alright Melvin. I've been really busy with wave work.

**Comment by dannymcclain** [comment]
Same as above Melv.

**Comment by dannymcclain** [comment]
We're going to get to it Melvin. It is OK!

**Comment by JmillsExpensify** [comment]
Helping out with this one, so I'm going to assign myself as well.

**Comment by dannymcclain** [comment]
We're letting search shake out before making any big changes to the preview component.

I'm going to leave this open for now just in case we have any brilliant ideas as we implement search, but I also think it would be acceptable to close this since we have a pretty jam packed roadmap of releases scheduled.

**Comment by JmillsExpensify** [comment]
I think we should close it. If it's a priority, we can come back to it.
